packages/cyberstorm/src/components/Markdown/Markdown.module.css

```diff
@@ -0,0 +1,145 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+  color: var(--color-text--default);
+  font: var(--font-body);
+  font-weight: 500;
+  line-height: 1.6rem;
+}
+
+/*
+  TODO: Use proper styles instead of these placeholder values. The
+  content rendered on this page should match how the content is actually
+  rendered on the package page. I don't think Figma contains all the
+  stuff supported by markdown, so this needs to be discussed further and
+  all the styles should be updated all at once.
+
+  We can't control the elements inside the Markdown wrapper, e.g. to
+  give them class names, and linting requires pure selectors, so repeat
+  the .root class name on each selector.
+*/
+.root h1,
+.root h2,
+.root h3,
+.root h4,
+.root h5,
+.root h6 {
+  display: flex;
+  gap: 0.5rem;
+  align-items: flex-start;
+  align-self: stretch;
+  padding: 2rem 0 0.5rem;
+  border-bottom: 1px solid #29295b;
+  font: var(--font-body);
+  font-weight: 700;
+  line-height: normal;
+}
```

Comment:
I edited the styles of headings to flow a bit more nice. I yoinked the bigger padding and border off and gave it only on h1 and h2, while setting `padding-top:1rem;` on the rest. You can add the lines below.

```
.root h1,
.root h2,
.root h3,
.root h4,
.root h5,
.root h6 {
  display: flex;
  gap: 0.5rem;
  align-items: flex-start;
  align-self: stretch;
  font: var(--font-body);
  font-weight: 700;
  line-height: normal;
}

.root h1,
.root h2 {
  padding: 2rem 0 0.5rem;
  border-bottom: 1px solid #29295b;
}

.root h3,
.root h4,
.root h5,
.root h6 {
  padding-top: 1rem;
}
```